content/app/app-dev-course/modul3/_index.nb.md

```diff
@@ -8,6 +8,13 @@ weight: 20
 
 {{% notice warning %}}
  Denne modulen krever at du er medlem av en organisasjon som har et etablert testmiljø for Altinn Apps. Dersom dette ikke er tilfellet kan du gå videre til [neste modul](../modul4/).
+ <br><br>
+ Hvis du har en personlig applikasjon som du vil flytte til en organisasjon, 
+ [så kan du spørre en administrator eller service desk](https://docs.altinn.studio/nb/app/getting-started/create-user/#bli-del-av-en-organisasjon).
+
```

Comment:
Let's use relative links so they don't navigate to docs.altinn.studio when running the docs locally
```suggestion
 [så kan du spørre en administrator eller service desk](/nb/app/getting-started/create-user/#bli-del-av-en-organisasjon).

```